android/app/src/main/java/com/mattermost/helpers/database_extension/Thread.kt

```diff
@@ -17,7 +18,16 @@ internal fun insertThread(db: WMDatabase, thread: ReadableMap) {
         val lastViewedAt = try { thread.getDouble("last_viewed_at") } catch (e: NoSuchKeyException) { 0 }
         val unreadReplies = try { thread.getInt("unread_replies") } catch (e: NoSuchKeyException) { 0 }
         val unreadMentions = try { thread.getInt("unread_mentions") } catch (e: NoSuchKeyException) { 0 }
-        val lastReplyAt = try { thread.getDouble("last_reply_at") } catch (e: NoSuchKeyException) { 0 }
+        val lastReplyAt = try {
+            var v = thread.getDouble("last_reply_at")
+            if (v == 0.0) {
+                val post = thread.getMap("post")
+                if (post != null) {
+                    v = post.getDouble("create_at")
+                }
+            }
+            v
```

Comment:
Used in many places, can be a function probably

Reply:
👍🏽